img/index.html

```diff
@@ -0,0 +1,302 @@
+  <link rel="stylesheet" href="css/main.css" />
+  <title>Hab Asm Portfolio</title>
+</head>
+<body>
+  <div class="main_holder">
+    <header class="header white_background">
+      <nav class="navbar flex">
+        <a href="#" class="navbar__link link size-20 weight-700 line_height-20">HABASM</a>
+        <div class="navbar__menu">          
+        </div>
+        <img class="navbar__hamb" src="images/hamburguer.png" alt="hamburguer icon" />
+      </nav>
+    </header>
+    <section class="head flex">
+      <h1 class="titleview size-40 weight-700 line_height-52">
+        I'm HABASM <span>Glad to see you!</span>
+      </h1>
+      <p class="description size-16 weight-400 line_height-24">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact
+        me.
+      </p>
+      <a href="#" class="head__link link size-16 weight-500 line_height-24 clr-p-400">My Projects</a>
+      <ul class="head__icons ul flex">
+        <li class="head__icons__li">
+          <a href="https://twitter.com/AsimareHabitamu" class="head__icons__link link"><img src="images/twitter.png" alt="twitter icon" /></a>
+        </li>
+        <li class="head__icons__li">
+          <a href="https://www.linkedin.com/in/habitamu-asimare-aa824b169/" class="head__icons__link link"><img src="images/Linkedin.svg" alt="linkedin icon" /></a>
+        </li>
+        <li class="head__icons__li">
+          <a href="#" class="head__icons__link link"><img src="images/M.png" alt="M icon" /></a>
+        </li>
+        <li class="head__icons__li">
+          <a href="https://github.com/HabAsm" class="head__icons__link link"><img src="images/github.svg" alt="github icon" /></a>
+        </li>
+        <li class="head__icons__li">
+          <a href="#" class="head__icons__link link"><img src="images/hand.svg" alt="hand icon" /></a>
+        </li>
+      </ul>
+    </section>
+
+    <section class="works" id="works">
+      <ul class="cards ul grid">
+        <li class="card flex white_background">
+          <img class="card__img" src="images/tonic-project-desktop.svg" alt="tonic project" />
+          <div class="card__text">
+            <h2 class="card__title weight-700 size-32 line_height-44 clr-n-800">
+              Tonic
+            </h2>
+            <div class="card__subtitle flex">
+              <p class="card__subtitle-1 weight-600 size-13 line_height-16 clr-n-600">
+                CANOPY
+              </p>
+              <ul class="card__subtitle-ul flex">
+                <li class="card__subtitle-li weight-600 size-13 line_height-16 clr-n-500">
+                  Back End Dev
+                </li>
+                <li class="card__subtitle-li weight-600 size-13 line_height-16 clr-n-500">
+                  2015
+                </li>
```

Comment:
Great job💪 on changing the color of the bullet points as requested by the previous reviewer🙄. However, please 🙏remove the extra button on the first card and adjust the spacing between the bullet points. This will enhance the overall appearance of your project and make it more similar to the original Figma design.🙏 KIndly refer to the screenshot below to compare your design with Figma's design.

## Your Design
![image](https://user-images.githubusercontent.com/63445083/227594440-7c36f938-a8ea-4658-8f41-d412be05ac33.png)

## Figma Design
![image](https://user-images.githubusercontent.com/63445083/227595182-bf7b3d11-3087-44c0-a806-87d2f1b2f898.png)